src/main/objects/Consts.js

```diff
@@ -113,7 +129,23 @@ var keyStrBase = {
     'A': BASE_NOTES.A,
     'Bb': BASE_NOTES.B,
     'B': BASE_NOTES.B,
-    'Cb': BASE_NOTES.C
+    'Cb': BASE_NOTES.C,
+
+    'c': BASE_NOTES.C,
```

Comment:
Te zmiany są po to, by ominąć to wywołanie toUpperCase?

Reply:
Tak. U mnie to this.key[0] = this.key[0].toUpperCase() nie działało w pełni poprawnie toteż na szybko zmieniłem to w ten sposób.

Reply:
Na razie niech zostanie, ale nie jest to zbyt eleganckie...